tldr/cross_file_calls/parsers/c.py

```
@@ -0,0 +1,157 @@
+            if arguments_node:
+                for i, child in enumerate(arguments_node.children):
+                    if child.type == ',':
+                        continue
+                    args.append({
+                        'position': i,
+                        'type': 'positional',
+                        'value': None,
+                        'name': None
+                    })
+        except Exception:
+            pass
```

Ruff (0.14.11)

`try`-`except`-`pass` detected, consider logging the exception

(S110)